### PR TITLE
U13 fixes

### PR DESCRIPTION
**Pull request by shubham7109**
This PR fixes the following errors:
- [x] Edit attachments: Delete confirmation buttons is not displayed
- [x] Edit with branch versioning: App crashes when the branch already exists instead of throwing error or warning
- [x] Generate offline map with local basemap: Selecting no to local basemap, retrieves an error message and job is not started 
- [x] Query map image sublayer: Query button was disabled

### CODE REVIEW

File: kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt [hunk 1]
```diff
@@ -227,15 +227,19 @@ class MainActivity : AppCompatActivity() {
     val serviceVersionInfoFuture =
       serviceGeodatabase.createVersionAsync(serviceVersionParameters)
     serviceVersionInfoFuture.addDoneListener {
-      // get the new version's name and switch to it
-      val serviceVersionInfo = serviceVersionInfoFuture.get()
-      createdVersionName = serviceVersionInfo.name
-      switchVersion(null)
+        try{
```
Comment (TADraeseke) [inline kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt:230]:
```suggestion
        try {
```

File: kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt [hunk 2]
```diff
@@ -227,15 +227,19 @@ class MainActivity : AppCompatActivity() {
...
+            // get the new version's name and switch to it
+            val serviceVersionInfo = serviceVersionInfoFuture.get()
+            createdVersionName = serviceVersionInfo.name
+            switchVersion(null)
+            // hide the create version button and allow the user to switch versions now
+            createVersionButton.visibility = View.GONE
+            switchVersionButton.visibility = View.VISIBLE
+            }catch (e: java.lang.Exception){
```
Comment (TADraeseke) [inline kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt:238]:
```suggestion
            } catch (e: java.lang.Exception) {
```

File: kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt [hunk 3]
```diff
@@ -227,15 +227,19 @@ class MainActivity : AppCompatActivity() {
...
+            Log.e(TAG, e.message.toString())
+            Toast.makeText(this, e.message.toString(),Toast.LENGTH_SHORT).show()
```
Comment (TADraeseke) [inline kotlin/edit-with-branch-versioning/src/main/java/com/esri/arcgisruntime/sample/editwithbranchversioning/MainActivity.kt:240]: Could be good to prepend with some plain English explanation of what might have gone wrong here. "Error getting service info: " + e.message, or something like that

**Review by TADraeseke — CHANGES_REQUESTED** [review]
Nice job fixing these. Just a few small comments below. Thanks @shubham7109!

**Comment by shubham7109** [comment]
Thanks Trevor for the feedback, I've gone ahead and added the  changes in!

**Review by TADraeseke — APPROVED** [review]
LGTM! Just one small formatting thing below. Thanks @shubham7109!

**Pull request merged**